scout/demo/643594.config.yaml

```diff
@@ -27,6 +27,11 @@ samples:
     mt_bam: scout/demo/reduced_mt.bam
     vcf2cytosure: scout/demo/ADM1059A2.dummy.cgh
     tissue_type: blood
+    chromograph_images:
+        'autozygous': 'scout/demo/images/chromograph_demo/autozigous_regions'
```

Comment:
Hm, sorry, didn't you just change those? all files are named "autozigous_". It doesn't really matter, but might look a tad weird?

Reply:
Fixed!